mdal/frmts/mdal_3di.cpp

```diff
@@ -194,7 +194,13 @@ std::set<std::string> MDAL::Driver3Di::ignoreNetCDFVariables()
   return ignore_variables;
 }
 
-void MDAL::Driver3Di::parseNetCDFVariableMetadata( int varid, const std::string &variableName, std::string &name, bool *is_vector, bool *is_x )
+void MDAL::Driver3Di::parseNetCDFVariableMetadata(
+  int varid,
+  const std::string &variableName,
+  std::string &name,
+  bool *is_vector,
+  bool *is_x,
+  std::map<std::string, std::string> & )
```

Comment:
add options explicitely and use MDAL_UNUSED in the function body https://github.com/lutraconsulting/MDAL/blob/417898347448e1dab7decc2d31a9c418873f9513/mdal/mdal_utils.hpp#L23